docs/pages/includes/enterprise/ent-user-prereqs.mdx

```diff
@@ -0,0 +1,30 @@
+<Tabs>
```

Comment:
If this partial is only going to be used in SSO guides, should we move it to the `docs/pages/includes/sso` directory?